vtex-apps/search-resolver
# Search resolver reliability

#### What problem is this solving?



#### How should this be manually tested?

[Check the new logs here](https://vtex.splunkcloud.com/en-US/app/vtex_io_apps/search?earliest=1662130800&latest=1662562934&q=search%20index%3Dio_vtex_logs%20app%3Dvtex.search-resolver%401.*%20production%3Dfalse%20%22Degraded%20search%22%20%7C%20timechart%20count%20by%20data.service&display.page.search.mode=fast&dispatch.sample_ratio=1&display.general.type=visualizations&display.page.search.tab=visualizations&sid=1662736142.20411495_0E315D9B-CDE1-4960-A650-072CF75459D7)

#### Checklist/Reminders


#### Screenshots or example usage

#### Type of changes


✔️ | Type of Change
---|---
_ | Bug fix 
_ | New feature 
_ | Breaking change 
✔️ | Technical improvements 

#### Notes
